command-line-interface.md

```diff
@@ -0,0 +1,246 @@
+# Operations
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
The README defines the protocols required for our various certifications.  This file is an API protocol.  If you fail this MUST, you fail to provide the command line API protocol.  If you fail the command line API protocol but *pass* an alternative API protocol (we don't have any others now, but may add more in the future), then you can still be certified as an “OCI Linux container runtime with the foo-bar API” (or whatever).  But you won't be certified as an “OCI Linux container runtime with the command-line API”.

I'd rather not qualify this particular requirement with “runtimes implementing a command line API” because this entire file falls under that condition (and the existing README language should make it clear).  If the existing README language (from #527) is not clear (it seems clear to me), then the README language should be updated.  But getting that right seems orthogonal to defining the spec protocol, so I'd rather have it handled in a separate PR.